stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListItemStyle.kt

```diff
@@ -105,6 +108,8 @@ public data class MessageListItemStyle(
     public val iconIndicatorRead: Drawable,
     public val iconIndicatorPendingSync: Drawable,
     public val iconOnlyVisibleToYou: Drawable,
+    public val iconFailedMessage: Drawable,
+    public val iconBannedMessage: Drawable,
```

Comment:
I would put these as the last 2 properties, just to avoid breaking API changes for our users (in case they're building the class or destructuring).

Reply:
Refactored so that the properties are the last two